src/main/java/com/socrata/api/SodaDdl.java

```diff
@@ -355,16 +352,9 @@ public ClientResponse issueRequest() throws LongRunningQueryException, SodaError
 
         try {
             final ClientResponse response = requester.issueRequest();
-            return mapper.readValue(response.getEntity(InputStream.class), AssetResponse.class);
-            //return response.getEntity(AssetResponse.class);
+            return response.getEntity(AssetResponse.class);
```

Comment:
jackson looks for json content-type in getEntity.  But the actual content type returned from the server is text/plain despite that it is json.  Using readValue from input stream get around this problem.